blaise82/l.m.s-server
# #170868440 set up mocha

#### What does this PR do?
- set up mocha

#### Description of Task to be completed?

#### How should this be manually tested?
- clone this repository
- check out to  ch-setup-mocha-170868440  branch
- run `npm run install`
- run `npm run test`

#### What are the relevant pivotal tracker stories?
#170868440 
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-01-23 at 17 03 14" src="https://user-images.githubusercontent.com/52036624/72995851-49f6f700-3e02-11ea-9f7f-673e9f83167b.png">

[finishes #170868440 ]
